app/views/docs/getting-started-for-server.phtml

```diff
@@ -55,6 +55,16 @@ $dartVersion = $versions['dart'] ?? '';
             <pre class="line-numbers"><code class="prism language-typescript" data-prism>import * as sdk from "https://deno.land/x/appwrite/mod.ts";</code></pre>
         </div>
     </li>
+    <li>
+        <h3>.NET</h3>
+        <div class="ide margin-top-small" data-lang="bash" data-lang-label="Deno SDK">
+            <pre class="line-numbers"><code class="prism language-bash" data-prism># Package Manager
+Install-Package Appwrite
+
+# or .NET CLI
+csharp add package Appwrite</code></pre>
```

Comment:
This should be `dotnet add package Appwrite`

Reply:
😄 Find and replace